prisma-fmt/src/get_config.rs

```diff
@@ -249,9 +382,9 @@ mod tests {
             }
         });
         let expected = expect![[
-            r#"{"message":"\u001b[1;91merror\u001b[0m: \u001b[1mEnvironment variable not found: DOES_NOT_EXIST.\u001b[0m\n  \u001b[1;94m-->\u001b[0m  \u001b[4mschema.prisma:5\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\u001b[1;94m 4 | \u001b[0m                url = env(\"DBURL\")\n\u001b[1;94m 5 | \u001b[0m                directUrl = \u001b[1;91menv(\"DOES_NOT_EXIST\")\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\nValidation Error Count: 1","error_code":"P1012"}"#
+            r#"{"config":{"generators":[],"datasources":[{"name":"thedb","provider":"postgresql","activeProvider":"postgresql","url":{"fromEnvVar":"DBURL","value":null},"directUrl":{"fromEnvVar":"DOES_NOT_EXIST","value":null},"schemas":[]}],"warnings":[]},"errors":[{"file_name":"schema.prisma","message":"\u001b[1;91merror\u001b[0m: \u001b[1mEnvironment variable not found: DOES_NOT_EXIST.\u001b[0m\n  \u001b[1;94m-->\u001b[0m  \u001b[4mschema.prisma:5\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\u001b[1;94m 4 | \u001b[0m                url = env(\"DBURL\")\n\u001b[1;94m 5 | \u001b[0m                directUrl = \u001b[1;91menv(\"DOES_NOT_EXIST\")\u001b[0m\n\u001b[1;94m   | \u001b[0m\n"}]}"#
         ]];
-        let response = get_config(&request.to_string()).unwrap_err();
+        let response = get_config(&request.to_string());
```

Comment:
Can you please add the snapshot of what the response would look like when we want to treat `getConfig` as fallible, by calling `diagnostics.to_result()`? Mostly, I'd like to make sure we snapshot `Validation Error Count: x`, because it accidentally got lost in a PR 1+ years ago, and it's faster to track these kind of slips in `prisma-engines` than in `prisma`

Reply:
Idea is, we'll finish formatting the errors in the cli from now on (follow up PR to prisma/prisma is coming).
So, `to_result` on the whole `Diagnostics` will not actually be used. For individual errors, we already have snapshots.

Moreover, `get_config` returns already serialized result - so there is nothing even to call on `to_result` on at this stage.

Reply:
Thanks for the clarification